setup.py

```diff
@@ -31,5 +31,6 @@
         'License :: OSI Approved :: Apache Software License',
         'Programming Language :: Python',
         'Programming Language :: Python :: 2.7'
+        'Programming Language :: Python :: 3.7'
```

Comment:
Might as well include 3.4, 3.5, 3.6 while you're at it since those are the most common versions at the moment.